modules/nvidia-gpu-admin-dashboard-installing.adoc

```diff
@@ -0,0 +1,124 @@
+
+* Red Hat OpenShift 4.11+
+* NVIDIA GPU operator
+* link:https://helm.sh/docs/intro/install/[Helm]
+
+
+.PROCEDURE
+
+Use the following procedure to deploy the OpenShift Console NVIDIA GPU plugin.
+
+. Add the Helm repository:
++
+[source,terminal]
+----
+$ helm repo add rh-ecosystem-edge https://rh-ecosystem-edge.github.io/console-plugin-nvidia-gpu
+----
++
+[source,terminal]
+----
+$ helm repo update
+----
+
+. Install the Helm chart in the default NVIDIA GPU operator namespace:
++
+[source,terminal]
+----
+$ helm install -n nvidia-gpu-operator console-plugin-nvidia-gpu rh-ecosystem-edge/console-plugin-nvidia-gpu
+----
++
+.Example output
++
+[source,terminal]
+----
+NAME: console-plugin-nvidia-gpu
+LAST DEPLOYED: Tue Aug 23 15:37:35 2022
+NAMESPACE: nvidia-gpu-operator
+STATUS: deployed
+REVISION: 1
+NOTES:
+View the Console Plugin NVIDIA GPU deployed resources by running the following command:
+
+$ oc -n {{ .Release.Namespace }} get all -l app.kubernetes.io/name=console-plugin-nvidia-gpu
+
+Enable the plugin by running the following command:
+
+# Check if a plugins field is specified
+$ oc get consoles.operator.openshift.io cluster --output=jsonpath="{.spec.plugins}"
+
+# if not, then run the following command to enable the plugin
+$ oc patch consoles.operator.openshift.io cluster --patch '{ "spec": { "plugins": ["console-plugin-nvidia-gpu"] } }' --type=merge
+
+# if yes, then run the following command to enable the plugin
+$ oc patch consoles.operator.openshift.io cluster --patch '[{"op": "add", "path": "/spec/plugins/-", "value": "console-plugin-nvidia-gpu" }]' --type=json
+
+# add the required DCGM Exporter metrics ConfigMap to the existing NVIDIA operator ClusterPolicy CR:
+oc patch clusterpolicies.nvidia.com gpu-cluster-policy --patch '{ "spec": { "dcgmExporter": { "config": { "name": "console-plugin-nvidia-gpu" } } } }' --type=merge
+
+----
++
+The dashboard relies mostly on Prometheus metrics exposed by the NVIDIA DCGM Exporter, but the default exposed metrics are not enough for the dashboard to render the required gauges. Thus, the DGCM exporter should be configured to expose a custom set of metrics, as shown here.
++
+[source,yaml]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "console-plugin-nvidia-gpu.fullname" . }}
+  labels:
+    {{- include "console-plugin-nvidia-gpu.labels" . | nindent 4 }}
+data:
+  dcgm-metrics.csv: |
+    DCGM_FI_PROF_GR_ENGINE_ACTIVE, gauge, gpu utilization.
+    DCGM_FI_DEV_MEM_COPY_UTIL, gauge, mem utilization.
+    DCGM_FI_DEV_ENC_UTIL, gauge, enc utilization.
+    DCGM_FI_DEV_DEC_UTIL, gauge, dec utilization.
+    DCGM_FI_DEV_POWER_USAGE, gauge, power usage.
+    DCGM_FI_DEV_POWER_MGMT_LIMIT_MAX, gauge, power mgmt limit.
+    DCGM_FI_DEV_GPU_TEMP, gauge, gpu temp.
+    DCGM_FI_DEV_SM_CLOCK, gauge, sm clock.
+    DCGM_FI_DEV_MAX_SM_CLOCK, gauge, max sm clock.
+    DCGM_FI_DEV_MEM_CLOCK, gauge, mem clock.
+    DCGM_FI_DEV_MAX_MEM_CLOCK, gauge, max mem clock.
+----
```

Comment:
This is a truncated ConfigMap yaml file, not a complete ConfigMap file, missing the annotations and labels.  Would be better to show a complete ConfigMap file or at least the default configMap file found in the nvidia-gpu-operator namespace.  It is also not clear if the user need to make any changes to the existing default configMap file, since it does include all the DCGM_ metrics shown above.

Reply:
Replaced with the yaml you sent me in Slack.